src/App.tsx

```diff
@@ -15,49 +17,106 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  const visibleGoods = [...goods];
+
+  visibleGoods.sort((prevGood, currentGood) => {
```

Comment:
nit: Use more explicit variable names for prevGood and currentGood, such as prevItem and currentItem, for better readability 📘